### ISSUE TITLE
Wireless Essentia Terminal not working with AE2-rv3 (1.7.10)

### ISSUE THREAD

**Issue by repo-alt** [Nividica/ThaumicEnergistics#374]
I tried with thaumicenergistics-1.1.3.0.jar and appliedenergistics2-rv3-beta-6.jar 

btw, I tried to build rv3 branch to look up into the issue, but got weird build errors. What AE2 version did you use to build rv3 branch?

**Comment by repo-alt** [comment]
ExtraCells universal wireless terminal is also not working with Essentia Terminal module, I'm not sure if it is the same issue.

**Comment by Nividica** [comment]
> What AE2 version did you use to build rv3 branch?

appliedenergistics2-rv3-beta-5-dev.jar

**Comment by Nividica** [comment]
Incase you need any other dependencies:
![image](https://user-images.githubusercontent.com/1585989/57052520-c682ed80-6c55-11e9-80c6-a48a9c8682ff.png)

**Comment by repo-alt** [comment]
Yes, thanks, I was able to build the branch some time ago (only after I unzipped MDK*.zip in the libs, but I guesss your Mekanism jar). Btw, do you know why IC2 API is necessary to build? Java gives weird errors w/o it but I did not find any usage of IC2 API in the code. (Same with RoraryCraft)

**Comment by Nividica** [comment]
My only guess on the IC2 is that it is a dependency, of a dependency. AE _can_ use IC2 power, but I don't think it was a hard dependency

**Comment by remclave** [comment]
As this bug is still open, the problem exists in conjunction with the appliedenergistics2-rv3-beta-6 version as well.

Version being used thaumicenergistics-1.1.3.0

Reported here: https://github.com/JadedPacks/Agrarian-Skies-2/issues/92

**Comment by MadProbe** [comment]
Any updates on this or is this mod abandoned?

**Comment by repo-alt** [comment]
I fixed this bug in our fork some time ago, but I'm not sure if it will work with vanilla AE2, you may try though: http://jenkins.usrv.eu:8080/job/Thaumic-Energistics/
